CONTRIBUTING.md

```diff
@@ -1,167 +1,113 @@
-﻿# CDNJS contributing document
+﻿# Contribute to CDNJS
 
- - Please read the instructions about sparseCheckout and shallow clone before any git clone operation to this huge repository: [documents/sparseCheckout.md](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md), especially if you are using MacOS which uses case-insensitive filesystem by default, will bring you other issues.
- - Please make sure you are not going to submit a personal library, we have popularity requirement for the libraries, for the detail, see: [issue](#b-request-a-new-library)
+* Read the [sparseCheckout documentation](/documents/sparseCheckout.md) before you clone the CDNJS repo locally. It is not recommended to clone the whole repo as is due to its large size. Using sparse-checkout also works around issues on case-insensitive file systems (HFS+ on macOS).
```

Comment:
Actually, sparseCheckout is about about the clone process, the shallow clone is. sparseCheckout is about not to checkout all the files.